src/mcp_agent/workflows/parallel/fan_in.py

```
@@ -122,24 +154,48 @@ async def generate_structured(
+                    request_params=request_params,
+                )
+
+                if self.context.tracing_enabled:
+                    try:
+                        span.set_attribute(
+                            "structured_response_json",
+                            structured_response.model_dump_json(),
+                        )
+                    # pylint: disable=broad-exception-caught
+                    except Exception:
+                        pass  # no-op for best-effort tracing
```

Ruff (0.11.9)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)